internal/common/common.go

```diff
@@ -33,6 +33,9 @@ const (
 	// LabelValueUSDEConfig is the label value used to identify the USDE ConfigMap
 	LabelValueUSDEConfig = "usde-config"
 
+	// LabelValueNamespaceConfig is the label value used to identify the namespace-level ConfigMap
```

Comment:
```suggestion
	// LabelValueNamespaceConfig is the label value used to identify the user's namespace-level ConfigMap
```

Reply:
feel free to re-word, but wanted to contrast from "numaplane-system" namespace for clarity